Portfolio/index.css

```diff
@@ -0,0 +1,772 @@
+  
+      margin-right: 34px;
+  
+  }
+  .text-left strong{
+    font-size: 5rem;
+    color: #00D8FF;
+  }
+  
+.btn{
+    color: #004572;
+    background-color: #00D8FF;
+    margin-left: 361px;
+    font-size: 2.5rem;
+    margin-top: 37px;
+     padding: 19px 55px;
+    border-radius: 32px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+     cursor: pointer;
+  
+}
+
+
+.myskills2 h1{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 4rem;
+    font-weight: 400;
+    font-family: Revalia;
+    color: #00D8FF;
+    line-height: 90px;
+    margin-right: -1260px;
+}
+
+
+  }
+ 
+
+
+  /* for under 789px */
+ 
+  @media only screen and (max-width: 789px){
+    
+* {
+    margin: 0px;
+    padding: 0px;
+    border: 0px;
+    
+    /* h1 {
+        margin-left: 10px;
+    } */
+
+}
+
+
+
+.image-container img {
+	width: 164px;
+    padding: 0px 0px;
+    margin-left: 29px;
+
+
+    
+}
+.image-container h2 {
+    font-size: 2.8rem;
+    margin: 0px 52px;
+    }
+
+    .myskills h1 {
+        justify-content: center;
+        align-items: center;
+        margin-left: -31px;
+    }
+    .myskills {
+        margin-left: 838px;
+        height: 92vh;
+    }
+
+    .John{
+        max-width: 100%;
+    }
+  }
+  
+/* contact */
+
+
+.contact{
+    font-family:Roboto;
+    /* display: flex; */
+    height: 300px;
+    padding: 20px 5px;
+    background-color:#003E66;
+}
+.GIT h1{
+    color: #00D8FF;
+}
+.GIT span{
+    color: #9ABEE9;
+}
+.GIT img{
+    padding: 0px 3px;
+    
+}
+.right input{
+    padding: 5px 10px;
+    margin: 5px 23px;
+    border-radius: 5px;
+    border: none;
+    width: 94%;
+    letter-spacing: 0.2rem;
+    color: black;
+    
+
+}
+.right textarea{
+    padding: 5px 10px;
+    letter-spacing: 0.2rem;
+    margin: 5px 23px;
+    color: #3F7090;
+    border: none;
+    border-radius: 5px;
+    width: 94%;
+}
+.right h2{
+    margin-left: 25px;
+}
+.submitbtn{
+    margin-left:350px;
+    background: transparent;
+    border:1px solid black;
+    border-radius: 10px;
+    cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 5px 20px;
+}
+.right h2{
+    color:  #003E66;
+}
+.right{
+    padding: 30px 0px;
+
+    float: right;
+    margin-right: 90px;
+    width: 68%;
+    border-radius: 20px;
+    line-height: 20px;
+    margin-top: 20px;
+    background-color: #9ABEE9;
+    color: black;
+
+}
+
+.left{
+    display: inline-block;
+    margin-left: 86px;
+    margin-top:50px;
+    letter-spacing: 19%;
+    word-spacing: 0.2rem;
+    font-weight: 400;
+    font-size: 20px;
+   
+}
+
+
+
+.lefticons img{
+    padding: 0px 2px;
+    margin-top: 4px;
+    
+    margin-left: -px;
+    gap: 15px;
+
+}
+
+
+
+
+/* footer */
+.footer{
+    /* display: inline-block; */
+    /* background-color: #f2faff; */
+    justify-content: center;
+    align-items: center;
+    background-color: #003E66;

```

Comment:
There seems to be a space between the negative sign and '1 px'. This may cause issues in CSS formatting and should be corrected.